### ISSUE TITLE
Unknown keys in service files

### ISSUE THREAD

**Issue by murraybd** [ubuntu/ubuntu-release-metrics#23]
I was looking at the log on the KPI collector system and found the following:

systemd[1]: /run/systemd/generator/run-metric-collector@versions.timer:8: Unknown key name 'FixedRandomDelay' in section 'Timer', ignoring.
systemd[1]: /run/systemd/generator/run-metric-collector@upload_queues.timer:8: Unknown key name 'FixedRandomDelay' in section 'Timer', ignoring.
systemd[1]: /run/systemd/generator/run-metric-collector@sponsoring.timer:8: Unknown key name 'FixedRandomDelay' in section 'Timer', ignoring.
systemd[1]: /run/systemd/generator/run-metric-collector@rls_bugs.timer:8: Unknown key name 'FixedRandomDelay' in section 'Timer', ignoring.
systemd[1]: /run/systemd/generator/run-metric-collector@images.timer:8: Unknown key name 'FixedRandomDelay' in section 'Timer', ignoring.
systemd[1]: /run/systemd/generator/run-metric-collector@autopkgtest.timer:8: Unknown key name 'FixedRandomDelay' in section 'Timer', ignoring.

**Comment by seb128** [comment]
Is that still an issue? Which Ubuntu version is installed on the server? (the key should be valid since systemd v247)

**Comment by murraybd** [comment]
Yes, it is still an issue. I guess we should upgrade the server?

`Dec 13 18:00:21 juju-640a8f-prod-ubuntu-release-kpi-collector-3 systemd[1]: /run/systemd/generator/run-metric-collector@cnf.tim
er:8: Unknown key name 'FixedRandomDelay' in section 'Timer', ignoring.
Dec 13 18:00:21 juju-640a8f-prod-ubuntu-release-kpi-collector-3 systemd[1]: /run/systemd/generator/run-metric-collector@autopkg
test.timer:8: Unknown key name 'FixedRandomDelay' in section 'Timer', ignoring.
ubuntu@juju-640a8f-prod-ubuntu-release-kpi-collector-3:~$ cat /etc/lsb-release 
DISTRIB_ID=Ubuntu
DISTRIB_RELEASE=20.04
DISTRIB_CODENAME=focal
DISTRIB_DESCRIPTION="Ubuntu 20.04.6 LTS"
ubuntu@juju-640a8f-prod-ubuntu-release-kpi-collector-3:~$ apt-cache policy systemd
systemd:
  Installed: 245.4-4ubuntu3.22
`

**Comment by seb128** [comment]
The key is ignored but it means the collectors trigger at the same for example after a restart, which is probably not a major issue / not important enough that we have to update the Ubuntu version on the system just to resolve it, it might still be a good idea to run a more recent LTS version though...

**Comment by Hyask** [comment]
Service has been migrated recently and now runs on Noble. Everything works fine, closing the issue.
